src/message/index.tsx

```diff
@@ -4,9 +4,35 @@ import { color, space, font } from '_utils/branding'
 import Message from './Message'
 
 const StyledMessage = styled(Message)`
-  & {
+  & blockquote {
     margin: 0;
+    display: inline-block;
+    padding-bottom: ${space.s}
+  }
+  
+  & {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  & .kirk-message-content {
+    word-break: break-word;
+  }
+  
+  & .kirk-message-content,
+  & .kirk-message-annotation {
+    display: flex;
+    margin-left: 48px;
```

Comment:
48px would be ${space.xxl}

Reply:
Done and I used the branding border radius too.